src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -1426,19 +1427,29 @@ def create_flavor(self, flavor: FlavorRequestModel) -> FlavorResponseModel:
                     f"'{flavor.user}' user."
                 )
 
-            new_flavor = FlavorSchema(
-                name=flavor.name,
-                type=flavor.type,
-                source=flavor.source,
-                config_schema=flavor.config_schema,
-                integration=flavor.integration,
-                project_id=flavor.project,
-                user_id=flavor.user,
-            )
-            session.add(new_flavor)
-            session.commit()
+            config_schema = json.dumps(flavor.config_schema)
```

Comment:
Just out of curiosity, is there a reason why we moved the `json.dumps` here? The `RequestModel` also has a dict instead of a str for the config schema.

Reply:
Having this a json in the FlavorResponseModel makes the serialization of that model very ugly, as you now have a string representation of a json within another json. This looks much cleaner and makes the dict immediatyl useable within python.